rules/xcodeproj.bzl

```diff
@@ -695,10 +753,25 @@ def _xcodeproj_impl(ctx):
         if _is_current_project_file(f)
     ]
 
+    # It's necessary to tell xcodegen what is the type ('debug'/'release')
```

Comment:
I'd suggest to keep it agnostic of `Debug` and `Release` if possible. Perhaps we can designate them all as `debug` inside of XcodeGen?

Reason being there's a lot more permutations to build

Reply:
I see, I guess we can just set them all to `none` in xcodegen? https://github.com/yonaskolb/XcodeGen/blob/master/Docs/ProjectSpec.md#configs